Anyway, I do not undertsnad this error in the SealModule.cc which calls the FW macro:

```
Reading cached build data
>> Local Products Rules ..... started
>> Local Products Rules ..... done
>> Entering Package HLTrigger/HLTcore
>> Creating project symlinks
>> Compiling edm plugin /data/CMS0/CMSSW_7_4_X_2015-01-12-0200/src/HLTrigger/HLTcore/plugins/SealModule.cc 
In file included from /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc481/cms/cmssw-patch/CMSSW_7_4_X_2015-01-12-0200/src/FWCore/Framework/interface/stream/EDAnalyzerAdaptor.h:32:0,
                 from /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc481/cms/cmssw-patch/CMSSW_7_4_X_2015-01-12-0200/src/FWCore/Framework/interface/stream/EDAnalyzerBase.h:26,
                 from /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc481/cms/cmssw-patch/CMSSW_7_4_X_2015-01-12-0200/src/FWCore/Framework/interface/stream/EDAnalyzer.h:28,
                 from /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc481/cms/cmssw-patch/CMSSW_7_4_X_2015-01-12-0200/src/FWCore/ParameterSet/interface/ParameterSetDescriptionFiller.h:37,
                 from /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc481/cms/cmssw-patch/CMSSW_7_4_X_2015-01-12-0200/src/FWCore/ParameterSet/interface/ParameterSetDescriptionFillerPluginFactory.h:25,
                 from /afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc481/cms/cmssw-patch/CMSSW_7_4_X_2015-01-12-0200/src/FWCore/ServiceRegistry/interface/ServiceMaker.h:22,
                 from /data/CMS0/CMSSW_7_4_X_2015-01-12-0200/src/HLTrigger/HLTcore/plugins/SealModule.cc:1:
/afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc481/cms/cmssw-patch/CMSSW_7_4_X_2015-01-12-0200/src/FWCore/Framework/src/MakeModuleHelper.h: In instantiation of 'static std::unique_ptr<T> edm::MakeModuleHelper<Base>::makeModule(const edm::ParameterSet&) [with T = HLTEventAnalyzerAOD; Base = edm::one::EDAnalyzerBase]':
/afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc481/cms/cmssw-patch/CMSSW_7_4_X_2015-01-12-0200/src/FWCore/Framework/src/WorkerMaker.h:85:149:   required from 'std::shared_ptr<edm::maker::ModuleHolder> edm::WorkerMaker<T>::makeModule(const edm::ParameterSet&) const [with T = HLTEventAnalyzerAOD]'
/data/CMS0/CMSSW_7_4_X_2015-01-12-0200/src/HLTrigger/HLTcore/plugins/SealModule.cc:24:39:   required from here
/afs/cern.ch/cms/sw/ReleaseCandidates/vol1/slc6_amd64_gcc481/cms/cmssw-patch/CMSSW_7_4_X_2015-01-12-0200/src/FWCore/Framework/src/MakeModuleHelper.h:39:44: error: cannot allocate an object of abstract type 'HLTEventAnalyzerAOD'
       std::unique_ptr<T> module{new T(pset)};

```

--- a/HLTrigger/HLTcore/interface/HLTEventAnalyzerAOD.h
+++ b/HLTrigger/HLTcore/interface/HLTEventAnalyzerAOD.h
@@ -12,7 +12,7 @@
  */
 
 #include "FWCore/Framework/interface/Event.h"
-#include "FWCore/Framework/interface/stream/EDAnalyzer.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "HLTrigger/HLTcore/interface/HLTConfigProvider.h"
 #include "DataFormats/Common/interface/TriggerResults.h"
@@ -24,13 +24,14 @@ namespace edm {
 //
 // class declaration
 //
-class HLTEventAnalyzerAOD : public edm::stream::EDAnalyzer<> {
+class HLTEventAnalyzerAOD : public edm::one::EDAnalyzer<edm::one::WatchRuns> {
   
  public:
   explicit HLTEventAnalyzerAOD(const edm::ParameterSet&);
   virtual ~HLTEventAnalyzerAOD();
   static void fillDescriptions(edm::ConfigurationDescriptions & descriptions);
 
+  virtual void endRun(edm::Run const &, edm::EventSetup const&) override;
   virtual void beginRun(edm::Run const &, edm::EventSetup const&) override;
   virtual void analyze(const edm::Event&, const edm::EventSetup&) override;
   virtual void analyzeTrigger(const edm::Event&, const edm::EventSetup&, const std::string& triggerName);

--- a/HLTrigger/HLTcore/interface/HLTEventAnalyzerRAW.h
+++ b/HLTrigger/HLTcore/interface/HLTEventAnalyzerRAW.h
@@ -12,7 +12,7 @@
  */
 
 #include "FWCore/Framework/interface/Event.h"
-#include "FWCore/Framework/interface/stream/EDAnalyzer.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "HLTrigger/HLTcore/interface/HLTConfigProvider.h"
 #include "DataFormats/Common/interface/TriggerResults.h"
@@ -24,13 +24,14 @@ namespace edm {
 //
 // class declaration
 //
-class HLTEventAnalyzerRAW : public edm::stream::EDAnalyzer<> {
+class HLTEventAnalyzerRAW : public edm::one::EDAnalyzer<edm::one::WatchRuns> {
   
  public:
   explicit HLTEventAnalyzerRAW(const edm::ParameterSet&);
   virtual ~HLTEventAnalyzerRAW();
   static void fillDescriptions(edm::ConfigurationDescriptions & descriptions);
 
+  virtual void endRun(edm::Run const &, edm::EventSetup const&) override;
   virtual void beginRun(edm::Run const &, edm::EventSetup const&) override;
   virtual void analyze(const edm::Event&, const edm::EventSetup&) override;
   virtual void analyzeTrigger(const edm::Event&, const edm::EventSetup&, const std::string& triggerName);

--- a/HLTrigger/HLTcore/interface/TriggerSummaryAnalyzerAOD.h
+++ b/HLTrigger/HLTcore/interface/TriggerSummaryAnalyzerAOD.h
@@ -13,7 +13,7 @@
 
 #include "DataFormats/HLTReco/interface/TriggerEvent.h"
 #include "FWCore/Framework/interface/Event.h"
-#include "FWCore/Framework/interface/stream/EDAnalyzer.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 namespace edm {
   class ConfigurationDescriptions;
@@ -22,7 +22,7 @@ namespace edm {
 //
 // class declaration
 //
-class TriggerSummaryAnalyzerAOD : public edm::stream::EDAnalyzer<> {
+class TriggerSummaryAnalyzerAOD : public edm::one::EDAnalyzer<> {
   
  public:
   explicit TriggerSummaryAnalyzerAOD(const edm::ParameterSet&);

--- a/HLTrigger/HLTcore/interface/TriggerSummaryAnalyzerRAW.h
+++ b/HLTrigger/HLTcore/interface/TriggerSummaryAnalyzerRAW.h
@@ -13,7 +13,7 @@
 
 #include "DataFormats/HLTReco/interface/TriggerEventWithRefs.h"
 #include "FWCore/Framework/interface/Event.h"
-#include "FWCore/Framework/interface/stream/EDAnalyzer.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 namespace edm {
   class ConfigurationDescriptions;
@@ -22,7 +22,7 @@ namespace edm {
 //
 // class declaration
 //
-class TriggerSummaryAnalyzerRAW : public edm::stream::EDAnalyzer<> {
+class TriggerSummaryAnalyzerRAW : public edm::one::EDAnalyzer<> {
   
  public:
   explicit TriggerSummaryAnalyzerRAW(const edm::ParameterSet&);

--- a/HLTrigger/HLTcore/plugins/HLTEventAnalyzerAOD.cc
+++ b/HLTrigger/HLTcore/plugins/HLTEventAnalyzerAOD.cc
@@ -53,6 +53,9 @@ HLTEventAnalyzerAOD::fillDescriptions(edm::ConfigurationDescriptions& descriptio
 }
 
 void
+HLTEventAnalyzerAOD::endRun(edm::Run const & iRun, edm::EventSetup const& iSetup) {}
+
+void
 HLTEventAnalyzerAOD::beginRun(edm::Run const & iRun, edm::EventSetup const& iSetup)
 {
   using namespace std;

--- a/HLTrigger/HLTcore/plugins/HLTEventAnalyzerRAW.cc
+++ b/HLTrigger/HLTcore/plugins/HLTEventAnalyzerRAW.cc
@@ -71,6 +71,9 @@ HLTEventAnalyzerRAW::fillDescriptions(edm::ConfigurationDescriptions& descriptio
 }
 
 void
+HLTEventAnalyzerRAW::endRun(edm::Run const & iRun, edm::EventSetup const& iSetup) {}
+
+void
 HLTEventAnalyzerRAW::beginRun(edm::Run const & iRun, edm::EventSetup const& iSetup)
 {
   using namespace std;

